plan.sh

```diff
@@ -1,14 +1,15 @@
 pkg_name=ruby
 pkg_origin=core
-pkg_version=2.5.8
+pkg_major=3.0
+pkg_version=3.0.1
```

Comment:
This should be 3.0.2 at this point, otherwise we have CVEs

Reply:
ruby is update to 3.0.2 version.
https://github.com/chef-base-plans/ruby/pull/5